.travis.yml

```diff
@@ -15,23 +15,24 @@ addons:
   chrome: stable
 
 cache:
-  yarn: true
+  directories:
+    - $HOME/.npm
 
 env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
+  - npm install -g npm@5
+  - npm config set spin false
```

Comment:
@mansona @sivakumar-kailasam and I discussed different aspects of this commit and our long-term approach to PRs, reviews and `yarn`/`npm` offline. We'll be working out ways for us to all stay on the same page as a team, but fundamentally miscommunications here were unintentional.